src/one/nio/serial/UuidSerializer.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015 Odnoklassniki Ltd, Mail.Ru Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package one.nio.serial;
+
+import java.io.IOException;
+import java.util.UUID;
+
+class UuidSerializer extends Serializer<UUID> {
```

Comment:
You're right... I thought UUID is not supported by default. There is no reason for the custom hand-written serializer.